### ISSUE TITLE
Update for Dawntrail version?

### ISSUE THREAD

**Issue by Kyofushin** [snooooowy/EurekaHelper#44]
Is there any update on whether this plugin will be updated for the Dawntrail version?

**Comment by snorux** [comment]
Yes, there's a PR open for dawntrail, I just haven't gotten around to merging it yet.

**Comment by XIV-Enjoyer** [comment]
👀

**Comment by ascobolonline** [comment]
Would really appreciate the update as well, as we just started an Eureka group in the fc again :) Thanks for the effort

**Comment by EmberWindsong** [comment]
Update would be amazing!

**Comment by Corollax** [comment]
👀

**Comment by Spaarky1978** [comment]
Thank you for your time and effort on this plugin. I noticed the fork was updated, but it doesn't look the the PR was processed, is there by chance a timeline of when this might happen? 

Once again thanks for this plugin, it is highly useful.

**Comment by XIV-Enjoyer** [comment]
It turns out the creator has unexpectedly passed away. RIP. This was such a great plugin, thank you for your support.

This is probably not possible to be updated and will need a new fork. :(

**Comment by snorux** [comment]
Am moving house + a new computer. I swear I'll update this

**Comment by XIV-Enjoyer** [comment]
The creator has been reborn!

**Comment by XIV-Enjoyer** [comment]
You ok over there? I hope moving into the new house didn't cause any issues. A lot of people get baited into a lower priced home, but then they find out it's infested with scholars and doctors, and it's too late to back out. It's why we deal with higher property taxes, to get away from them.
